pallets/amm/src/lib.rs

```diff
@@ -189,8 +189,8 @@ decl_module! {
 			let who = ensure_signed(origin)?;
 
 			ensure!(
-				!amount.is_zero(),
-				Error::<T>::CannotCreatePoolWithZeroLiquidity
+				amount >= MIN_POOL_LIMIT,
+				Error::<T>::InsufficientPoolAssetBalance
```

Comment:
Any Suggestions for a more descriptive Error?

Reply:
~ MinimalPoolLiquidityRequirementNotMet ?
We'll have something more verbose in the UI anyways but you should be able to know what happened with just error msg.